Game/Resources/Opening Scene/Opening Scene.json

```diff
@@ -41,7 +41,7 @@
 			"Material" : 
 			{
 				"albedo" : "Resources/Opening Scene/Cave/Textures/cave_albedo",
-				"metallic" : "Resources/Opening Scene/Cave/Textures/cave_metallic",
+				"metallic" : "Resources/Opening Scene/Cave/Textures/Ambient",
```

Comment:
Any particular reason why this texture was changed to something called 'Ambient'?

Reply:
Must've been a weird bug when importing the audio. Will try to reproduce and write an issue.